### DISCUSSION TITLE
Can I use pyodide as a kernel underneath jupyter_server?

### DISCUSSION THREAD

**Discussion by kdarby-cqg** [jupyterlite/jupyterlite#1407] [Q&A]
I know this is a strange question. I want to set up a development environment outside of the browser using a pyodide python kernel and vscode (or vscode for web(!)). Users would develop and do research in this environment, leveraging bigger cpus, and then deploy code to the production environment inside a browser.

Just wanted to know if this was possible, and if not, if there were any hints about what it might take.

Thanks!
Kevin

I also asked this in the pyodide discussion here: 

https://github.com/pyodide/pyodide/discussions/4843#discussioncomment-9694458

**Comment by jtpio** [comment]
Thanks @kdarby-cqg for the suggestion.

This idea sounds good, and looks similar to the one expressed in https://github.com/jupyterlite/pyodide-kernel/issues/3.

There doesn't seem to be ongoing work about this at the moment, but if you would like to give it a try please feel free!
